content/python/concepts/strings/strings.md

```diff
@@ -101,7 +101,16 @@ print(my_string)
 # Output: It's a lovely day!
 ```
 
-Python also has a series of non-printable characters that can modify strings. For example, `\n` will add a new line, and `\t` will add a tab:
+You could also get around this problem by wrapping strings containing `'` characters in double quotes:
```

Comment:
For Docs, we'll refrain from using "you." Let's rephrase this 🟢

Reply:
Thanks; understood. Will change this later today.